linux_os/guide/services/obsolete/tftp/tftpd_uses_secure_mode/rule.yml

```diff
@@ -44,21 +49,57 @@ references:
     stigid@rhel7: RHEL-07-040720
     stigid@rhel8: RHEL-08-040350
 
-ocil_clause: '"server_args" line does not have a "-s" option, and a subdirectory is not assigned'
+ocil_clause: |-
+{{%- if product in ["rhel7","ol7","rhel8","ol8","rhv4"] %}}
+    '"server_args" line does not have a "-s" option, and a subdirectory is not assigned'
+{{%- else %}}
+    'the "ExecStart" line does not have a "-s" option, and a subdirectory is not assigned'
+{{%- endif %}}
 
 ocil: |-
-    If TFTP is not installed, this is not applicable. To determine if TFTP is
-    installed, run the following command:
+    Verify the TFTP daemon is configured to operate in secure mode.
+
+    Check if a TFTP server is installed with the following command:
+
+    {{%- if product in ["rhel7","ol7","rhel8","ol8","rhv4"] %}}
     <pre>$ rpm -qa | grep tftp</pre>
     <br /><br />
-    Verify <tt>tftp</tt> is configured by with the <tt>-s</tt> option by
-    running the following command:
+    {{%- else %}}
+    $ sudo dnf list --installed tftp-server
```

Comment:
In the generated HTML SRG Mapping table, there is missing a vertical whitespace before and after this command.